docs/pages/guides/notifications-services.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Using Push Notifications
+description: Learn about different push notification services.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo apps can work with any notification service or capabilities offered by the operating system thanks to [config plugins](/config-plugins/introduction/). The following services provide purpose-built Expo integrations for implementing push notifications in your app:
+
+> The [`expo-notifications`](/versions/latest/sdk/notifications/) library is designed and tested to work with Expo's push notification service. Some advanced features may not be compatible with third-party providers, as they often have their own native and React Native SDKs optimized for their services.
+
+## Expo push notifications
+
+[Expo Notifications](/versions/latest/sdk/notifications/) provides a unified API for handling push notifications across Android and iOS. It integrates seamlessly with your Expo account and is free to use.
+
+#### Key capabilities:
+
+- Fully compatible with the [`expo-notifications`](/versions/latest/sdk/notifications/) library
+- Includes an Expo Dashboard to track notifications and user engagement
+- Supports testing notifications with the [Expo Notifications Tool](https://expo.dev/notifications)
+
+#### Considerations and limitations:
+
+- Does not support advanced platform features like notification extensions
```

Comment:
re notification extensions:

I believe that if people want to have a notification extension, they can have it. To my knowledge, we're not preventing them from doing that. Listing this as a limitation is inaccurate. I can test it later to be sure but I tend to think this should be removed.

Reply:
I've definitely heard of people doing it, and it was more like "we can't guarantee this works". If we test it and it works, that would be great to note - not that we're suddenly supporting it, but we could adjust this message to something like "the library doesn't have anything built-in for this, it's not supported, but nothing is stopping you"

Reply:
Notification extensions are integrated pretty deeply into iOS, and I think we can guarantee they work to the same extent as Apple :D. But I will double-check next week. I think we can say that "iOS notification extension is supported but you need to set it up yourself /according to this guide/. (we don't have a guide for that). But saying they are not supported is not the right message.